docs/src/content/docs/api/events_linux.md

```
@@ -0,0 +1,43 @@
+---
+title: Events (Linux)
+tableOfContents:
+  maxHeadingLevel: 4
+sidebar:
+  order: 120
+---
+
+## Application Events
+
+#### `events.Linux.ApplicationStartup`
```

Markdownlint (0.35.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)